charts/agents-graphql-api/values.yaml

```diff
@@ -0,0 +1,62 @@
+# Default values for agents-graphql-apo.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from k8s-at-home common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+controller:
+  # -- Number of desired pods.
+  # We use 3 minimum to assure no outage durring rollout/preemtible node restarts
+  replicas: 3
+
+image:
+  # -- image repository
+  repository: "eu.gcr.io/logical-sled-220910/strg-at/agents-graphql-api"
+  # -- image tag
+  tag: ""
+  # -- image pull policy
+  # pullPolicy: IfNotPresent
+
+securityContext:
+  capabilities:
+    drop:
+      # -- drop all privileges as we dont need them
+      - ALL
+  # -- set root fs to read only for security reasons
+  readOnlyRootFilesystem: true
+  # -- do not allow privilege escalation for security reasons
+  allowPrivilegeEscalation: false
+  # -- do not run as root for security reasons
+  runAsNonRoot: true
+  # -- run as user with <id>
+  runAsUser: 1000
+
+env:
+  # -- The timezone in the container
+  TZ: UTC
+  # -- The application port where it will be acessible
+  PORT: "3000"
+  # -- The default node environment
+  NODE_ENV: "production"
+  # -- host for graphql
+  HOST: ""
+  # -- base url for authetication service
+  BASE_URL: ""
+
+service:
+  main:
+    primary: true
+    ports:
+      http:
+        port: 3000
+
+persistence:
+  config:
+    enabled: true
+    type: configMap
+    name: agents-graphql-api-configmap
+    mountPath: /app/config.yaml
```

Comment:
@tyriis  Currently in graphql is config hardcoded as /app/config.yaml. Will we change it later to env variable? If yes then of course I can set path also as /config here https://github.com/strg-at/agents-graphql-api/blob/main/src/index.js#L23

Reply:
as pointed out, this will not work, as it will now create a folder /app/config.yaml where it mounts the content of the configmap (each key a file) you can use the custom mount if you want to achieve it to have it in /app. see: https://docs.k8s-at-home.com/our-helm-charts/common-library-storage/#multiple-subpaths-for-1-volume

Reply:
@dusanscensny your way will work aswel